integration-tests/resteasy-jackson/src/main/java/io/quarkus/it/resteasy/jackson/GreetingResource.java

```diff
@@ -23,4 +23,12 @@ public String config(@Context ResteasyConfiguration config) {
         // test that configuration can be obtained from application.properties
         return config.getParameter("resteasy.gzip.max.input");
     }
+
+    @Path("quarkus-config")
+    @GET
+    @Produces("text/plain")
+    public String quarkusConfig(@Context ResteasyConfiguration config) {
+        // test that Quarkus configuration can be obtained from application.properties
+        return config.getParameter("quarkus.resteasy.gzip.max-input");
```

Comment:
I don't understand why we would test that given RESTEasy has no knowledge of Quarkus. So I don't expect this value to be available in the RESTEasy context.

What we want to check is that RESTEasy correctly understood the Quarkus config and is properly configured.

Reply:
> What we want to check is that RESTEasy correctly understood the Quarkus config and is properly configured.

- `testConfig` tests that property that is used by RESTEasy is configured
- `testQuarkusConfig` tests that:
  - property is available at runtime
  - property has lesser priority than RESTEasy-specific property because if it didn't, `testConfig` would fail (as it failed when I didn't do RESTEasy specific condition)

I can add more tests, what do you want to test, sending of load bigger than max and receiving 413? I didn't do it as I can see QE tests failing.

Reply:
Sure, I removed the test.